doc/source/user_guide/io.rst

```diff
@@ -110,11 +110,12 @@ names : array-like, default ``None``
   List of column names to use. If file contains no header row, then you should
   explicitly pass ``header=None``. Duplicates in this list will cause
   a ``UserWarning`` to be issued.
-index_col :  int or sequence or ``False``, default ``None``
-  Column to use as the row labels of the ``DataFrame``. If a sequence is given, a
-  MultiIndex is used. If you have a malformed file with delimiters at the end of
-  each line, you might consider ``index_col=False`` to force pandas to *not* use
-  the first column as the index (row names).
+index_col : int/string or sequence of int/string or ``False``, default ``None``
+  Column(s) to use as the row labels of the ``DataFrame``, either given as string name or column index.
+  If a sequence of int/string is given, a MultiIndex is used.
+  Columns used for the index (row names) are dropped from the actual columns of the input dataframe.
+  They are accessible via ``.index``.
```

Comment:
this is not relevant here.

Reply:
Just the line about indexes being accessible via `.index`, or also the one about them not being accessible as regular columns?

Reply:
@jreback?

Reply:
L166 and L117 can probably be removed.